### PR TITLE
Grammar review, second run

### PR DESCRIPTION
**Pull request by pavolloffay**
## Do not merge, this is only for grammar review

Current thesis pdf: https://github.com/hawkular/hawkular-datamining/raw/doc/doc/thesis.pdf

### CODE REVIEW

File: doc/content.tex [hunk 1]
```diff
@@ -1,1402 +0,0 @@
...
-    The problem was that Inventory is developed by another party and any concrete document
-    describing the integration was not proposed. The design of the integration was understood differently
-    by both parties, which lead to reimplementing of some parts. Therefore it is better to precisely define the
-    integration interface with other modules or projects.
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Evaluation of Implemented Models} \label{chap:evaluation}
-Evaluation process compares predictive capabilities of the implemented models to statistics produced by similar models
-from R language. Models from R language are selected because they are widely used by forecasters all around the
-world and de facto standard for statistical science \cite{kleiber2008applied}.
```
Comment (grammar-review) [inline doc/content.tex:1065]: ...,therefore,... (spojit vety a ciarka z oboc stran, ako som pisala v komente numero 9)

Comment (grammar-review) [inline doc/content.tex:1065]: tu by som kludne dala aj : To be more specific, ... / pridat "more" a kludne aj ciarku

Comment (grammar-review) [inline doc/content.tex:1065]: ..parameters,.. /ciarka

Comment (grammar-review) [inline doc/content.tex:1065]: neznelo by Ti lepsie?: In this work model, testing parameter is set to...

Comment (grammar-review) [inline doc/content.tex:1065]: Methodology used...

Comment (grammar-review) [inline doc/content.tex:1065]: Most importantly, ... / ciarka.

Comment (grammar-review) [inline doc/content.tex:1065]: It should evaluate forecast accuracy using genuine forecasts instead...

File: doc/appendix.tex [hunk 2]
```diff
@@ -1,158 +0,0 @@
-\chapter{Diagrams}
```
Comment (pavolloffay) [inline doc/appendix.tex]: test komentar

File: doc/content.tex (hunk 1 above)
Comment (grammar-review) [inline doc/content.tex:1065]: ..., therefore...

Comment (grammar-review) [inline doc/content.tex:1065]: ..., therefore, ... /ciarka

File: doc/appendix.tex (hunk 2 above)
Comment (pavolloffay) [inline doc/appendix.tex:1]: Testovaci komentar. Vidis ho Janka?

File: doc/content.tex (hunk 1 above)
Comment (grammar-review) [inline doc/content.tex:1065]: testovaci komentar

File: doc/appendix.tex [hunk 3]
```diff
@@ -1,158 +0,0 @@
...
-    \begin{figure}[H]
-        \begin{center}
-            % original from idea
-            %\scalebox{0.34}[0.475]{\includegraphics{img/src/maven-deps-tree.pdf}}
-            \scalebox{0.6}{\includegraphics{img/src/maven-deps-tree.pdf}}
```
Comment (pavolloffay) [inline doc/appendix.tex:6]: Test 2

File: doc/appendix.tex (hunk 2 above)
Comment (grammar-review) [inline doc/appendix.tex:1]: test

File: doc/appendix.tex [hunk 4]
```diff
@@ -1,158 +0,0 @@
...
-            \caption{Dependency tree of maven artifacts.}
-            \label{appen:maven-deps}
-        \end{center}
-    \end{figure}
-
```
Comment (grammar-review) [inline doc/appendix.tex:11]: tralala

File: doc/content.tex (hunk 1 above)
Comment (grammar-review) [inline doc/content.tex:1065]: test

File: doc/content.tex [hunk 5]
```diff
@@ -1,1402 +0,0 @@
...
-    During the implementation a lot of time was spent on integration with other modules, with Inventory in particular.
-    The problem was that Inventory is developed by another party and any concrete document
-    describing the integration was not proposed. The design of the integration was understood differently
-    by both parties, which lead to reimplementing of some parts. Therefore it is better to precisely define the
-    integration interface with other modules or projects.
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Evaluation of Implemented Models} \label{chap:evaluation}
-Evaluation process compares predictive capabilities of the implemented models to statistics produced by similar models
-from R language. Models from R language are selected because they are widely used by forecasters all around the
```
Comment (pavolloffay) [inline doc/content.tex:1064]: Testovaci komentar

Comment (grammar-review) [inline doc/content.tex:1064]: tiez

File: doc/content.tex (hunk 1 above)
Comment (grammar-review) [inline doc/content.tex:1065]: test test

Comment (grammar-review) [inline doc/content.tex:1065]: test test test

File: doc/appendix.tex [hunk 6]
```diff
@@ -1,158 +0,0 @@
...
-    \begin{figure}[H]
-        \begin{center}
```
Comment (grammar-review) [inline doc/appendix.tex:13]: test

File: doc/content.tex (hunk 1 above)
Comment (grammar-review) [inline doc/content.tex:1065]: test

**Pull request closed without merging**